backend/src/sync.js

```diff
@@ -454,7 +439,7 @@ async function syncGenesisState() {
     const genesisRecords = await retry(10, () =>
       nearRpc.sendJsonRpc("EXPERIMENTAL_genesis_records", [pagination])
     );
-    await saveGenesis(parsedGenesisTime, genesisRecords.records, offset);
+    await saveGenesis(genesisTime, genesisRecords.records, offset);
```

Comment:
why is it indexed by genesis time instead of genesis height?

Reply:
We need genesis time recorded and display on the frontend. We use genesis height other place to check and query blocks.

Reply:
It is not indexed by that. We use genesis time to indicate the creation time of Accounts.